include/items.h

```diff
@@ -1,4 +1,39 @@
-typedef enum {
+#define ITEM_HEAD_START 0xC3
+#define ITEM_ARMOR_START 0xA9
+#define ITEM_CAPE_START 0xD9
```

Comment:
These should use the enum values like `#define ITEM_CAPE_START ITEM_NO_CAPE`

Reply:
Done